content/en/quickstart/create-device.md

```diff
@@ -11,3 +11,105 @@ menu:
     weight: 10
 toc: true
 ---
+This guide will cover how to get the first steps while using the plgd cloud in the Raspberry Pi. This includes how to create cloud server (OCF Device) available from IoTivity-Lite open source project on the Github.
+
+## Requirements
+You will need the following: 
+- Raspberry Pi board with a microSD card with [Raspberry Pi OS](https://www.raspberrypi.org/software/operating-systems/#raspberry-pi-os-32-bit)
+- Optional: HDMI monitor and cable, USB keyboard and mouse for the Pi
+- Ethernet or Wi-Fi connection between a DHCP-enabled network and the Pi
+- Personal computer(Linux/MacOS recommended) on the same network as the Pi
+
+## Setting up the Raspberry Pi
+1. Set up your SD card as described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/2)
+2. Connect your Raspberry Pi as described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/3)
+3. Start up your Raspberry Pi as described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/4)
+4. Configuring your Raspberry Pi described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-using/9)
+    
+    #### Configure SSH(Remote Access Setting)
+    You make sure to enable SSH to allow remote access to your Raspberry Pi from another computer using SSH. 
```

Comment:
It's not required.
Make a note, not a separate question.
(check how note shortcode looks like in architecture/component-overview.md)
Enabling SSH is highly recommended as it simplifies access to device and fast adoption of the code if required. More information available [here]()